smoke-tests/images/servlet/build.gradle.kts

```diff
@@ -157,21 +157,47 @@ fun configureImage(
   val image = "ghcr.io/$repo/smoke-test-servlet-$server:$version-jdk$jdk$vmSuffix$platformSuffix-$extraTag"
 
   val jdkImage = if (vm == "hotspot") {
-    if (jdk == "24") {
+    if (jdk == "26-ea") {
```

Comment:
this isn't tested, but we could add it in the future, the point here is that this branch only supports -ea releases